docs/source/enterprise-features.mdx

```diff
@@ -63,13 +64,63 @@ Licenses are served via [Apollo Uplink](/federation/managed-federation/uplink/),
 
 A router instance's license is valid for the duration of your organization's current subscription billing period (plus a [grace period](#grace-period-for-expired-plans)), even if the router temporarily becomes disconnected from GraphOS.
 
+### Offline Enterprise license
+
+<Tip>
+
+Offline Enterprise license support for Apollo is available on an as-needed basis. It must be enabled on your Studio organization. For access, send a request to your Apollo contact.
```

Comment:
I think the "For access" sentences makes sense earlier. "It must be enabled on your Studio org" seems like an implementation detail.

```suggestion
Offline Enterprise license support is available on an as-needed basis. Send a request to your Apollo contact to enable it for your GraphOS Studio organization.
```